src/components/SvgVennDiagram2.tsx

```diff
@@ -158,29 +161,33 @@ export const SvgVennDiagram2 = ({ selectors }: Props) => {
         can filter the genes: if nothing is selected, all genes are included.
```

Comment:
The sentence `You can filter the genes: if nothing is selected, all genes are included.` should be removed if nucs are selected.